### ISSUE TITLE
Fairlearn Dashboard sensitive feature selection does not carry over from model comparison view to individual model view

### ISSUE THREAD

**Issue by romanlutz** [fairlearn/fairlearn#524]
#### Describe the bug
In the model comparison view we can select a sensitive feature and change that selection using a dropdown:
![image](https://user-images.githubusercontent.com/10245648/87904364-fe20b580-ca12-11ea-82b8-f5639b5cc8a5.png)

However, when switching to the individual model view by clicking on a model point in the chart the selection does not carry over. The same applies when changing the selection and switching back to the comparison view.

#### Expected Results
The selection should carry over.

@chisingh FYI

**Comment by romanlutz** [comment]
Closing this since we're not changing the dashboard anymore before removing it. This will be addressed by `raiwidgets`

**Comment by MarsLabAtSLU** [comment]
> Closing this since we're not changing the dashboard anymore before removing it. This will be addressed by `raiwidgets`

raiwidgets is just been launched Nov, 2 2020. I recommend we could still fix it and have version control take care of rest. Might be when raiwidgets gets released we can close this issue and document.

**Comment by romanlutz** [comment]
@MarsLabAtSLU It's just an "empty release" to reserve the name. We've already added a warning to the Fairlearn repo to indicate to users that it's going away, and the v0.5.0 release which is planned for tomorrow will be the last one to have the dashboard.

As I explained above, the dashboard was rolled back to the previous ipywidgets-based version and does not show this problem anymore. Rolling it forward to the flask-based version is theoretically possible, but we're removing it before the next release. The only place where the fix may be useful is in the new repository. I'm afraid I can't make the repo public until we have the initial release ready, but if you have an idea for how to fix it I'm interested since we'll encounter the same issue in `raiwidgets` which will be flask-based.
